File: stdlib/go/System.go
package main

import (
	"fmt"
	"os"
	"os/exec"
)

func runcmd(command dynamic) {

	cmd := exec.Command(command.(string))

	cmd.Run()
}

func shutdown(exitcode dynamic) {
	os.Exit(exitcode.(int))
}

func varTrace(vari dynamic) {
	fmt.Println(vari)
}

File: stdlib/js/System.js
var exec = import('child_process').exec;

class System {

    static runcmd(cmd) {
        exec(cmd,
            null
        );
    }

    static shutdown(exitcode) {
        throw new Error();
    }

    static varTrace(vari) {
        console.log("" + vari);
    }

}